PROJECT

```diff
@@ -0,0 +1,24 @@
+domain: service.binding
+layout:
+- go.kubebuilder.io/v3
+multigroup: true
+projectName: service-binding-operator
+repo: github.com/kubernetes-sigs/service-binding
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: service.binding
+  kind: ServiceBinding
+  path: github.com/kubernetes-sigs/service-binding/apis/v1alpha2
+  version: v1alpha2
+- api:
+    crdVersion: v1
+    namespaced: true
```

Comment:
should not be namespaced

Reply:
Fixed. 295e16d4ceb409b484b4899341ed97bd8dc65274